crates/cairo-lint-core/tests/enum_variant_names/mod.rs

```diff
@@ -76,6 +76,23 @@ enum Cake {
 }
 "#;
 
+const ALLOW_ENUM_WITH_PREFIX: &str = r#"
+#[allow(enum_variant_names)]
+enum Cake {
+   DoubleBlackForest,
+   DoubleHummingbirdForest,
+   DoubleBattenbergForest,
+}
+"#;
+
+const ALLOW_ENUM_WITH_SUFFIX: &str = r#"
+#[allow(enum_variant_names)]
+enum Cake {
+    ForestBlack,
+    ForestWhite,
+}
+"#;
```

Comment:
This one's testing a suffix, but if im not wrong, this case actually tests prefixes, right? Can we adjust that?